.gitignore

```diff
@@ -11,7 +11,7 @@ api_keys.py
 ### Data Files ###
 final_jsons/*.json
 shape_files/*.json
-*.csv
+final_jsons/*.csv
```

Comment:
Do we want to exclude csvs in just the one folder? or we can manually add csvs we want to track

Reply:
I'm cool with manually adding the csv but I couldn't figure out how to do it from sourcetree. I'll look around and see if I can figure it out or I'll just do it from the command line

Reply:
I do it from the command line. Just think it keeps the gitignore clearer with fewer lines and exceptions.